api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticsearchService.scala

```diff
@@ -69,6 +70,35 @@ class ElasticsearchService(elasticClient: ElasticClient)(
         }
     }
 
+  def executeMultiSearchRequest(
+    request: MultiSearchRequest): Future[Either[ElasticError, List[SearchResponse]]] =
+    spanFuture(
+      name = "ElasticSearch#executeQuery",
+      spanType = "request",
+      subType = "elastic",
+      action = "query"
```

Comment:
How would we tell the difference between these queries and the other ones? This span seems to be exactly the same as the other one.

Reply:
Good point, how about `spanType = "multiRequest"`?

Reply:
Makes sense, we might need to relook at these anyway as I am not clear what's being recorded in APM.
This will at least let us see if things are super wrong.

Reply:
This sort of labelling (eg `multiRequest`) isn't what `spanType` is used for currently - it's a poorly documented field which has some special meanings for APM eg https://www.elastic.co/guide/en/apm/agent/java/current/public-api.html#api-transaction-set-type.

The current pattern, and what I'd prefer, would be to update `name` to `ElasticSearch#executeMultisearchRequest` or something like that. I _think_ it's OK to change `spanType` but just to flag that it's not necessarily as simple as free text, and we already distinguish between the spans using `name`.